distribution/docs/src/main/resources/_contents/_tables/conf-org.codice.ddf.catalog.security.policy.xml.XmlAttributeSecurityPolicyPlugin-table-contents.adoc

```diff
@@ -0,0 +1,4 @@
+|<<org.codice.ddf.catalog.security.policy.xml.XmlAttributeSecurityPolicyPlugin,XML Attribute Security Policy Plugin>>
+|org.codice.ddf.catalog.security.policy.xml.XmlAttributeSecurityPolicyPlugin
+|XML Attribute Security Policy Plugin
```

Comment:
In `XmlResponseQueueTransformer-table-contents.adoc` you used "Xml" and here you're capitalizing it. I believe this is the capitalization format we want to use for this initialization, but I mostly care we're consistent.